src/Uno.Extensions.Templates/content/unoapp-extensions/build/steps/mobile.yml

```diff
@@ -0,0 +1,75 @@
+parameters:
+- name: major
+  type: string
+  default: '0'  
+- name: minor
+  type: string
+  default: '0'
+- name: package_id
```

Comment:
This parameter shouldn't be required since it can be inferred from the target framework